pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -621,11 +621,17 @@ private void commitSegmentMetadataInternal(String realtimeTableName,
     preProcessNewSegmentZKMetadata();
 
     // Step-2: Create new segment metadata if needed
-    LOGGER.info("Creating new segment metadata with status IN_PROGRESS: {}", committingSegmentName);
+    LOGGER.info("Creating new segment metadata with status: {}", Status.IN_PROGRESS);
     long startTimeNs2 = System.nanoTime();
     String newConsumingSegmentName =
         createNewSegmentMetadata(tableConfig, idealState, committingSegmentDescriptor, committingSegmentZKMetadata,
             instancePartitions);
+    if (newConsumingSegmentName != null) {
+      LOGGER.info("Created new segment metadata for segment: {} with status: {}.", newConsumingSegmentName,
+          Status.IN_PROGRESS);
+    } else {
+      LOGGER.info("Skipped creation of new segment metadata as new consuming segment name is: null");
+    }
```

Comment:
Suggest moving these logs into `createNewSegmentMetadata()` where we have better context. We don't create new consuming segment because the ingestion is paused